floodviz/map_utils.py

```diff
@@ -36,11 +65,29 @@ def site_dict(site_list, url_prefix):
 
 def write_geojson(filename, data):
     """Writes site data to a .json file so it can be mapped
-
        Args:
-           filename: the file to be written to
-           data: the data to be written to the file
+           filename: A string naming the file to be written to
+           data: A list of dicts with data to be written to the file
     """
+    if not type(filename) == str:
```

Comment:
These kind of checks are overkill. The user of this function knows to pass in a string and it does not need to end with .json.